src/modules/commentsApi.js

```diff
@@ -0,0 +1,23 @@
+export const addComments = async (itemId, userName, userComment) => {
+  const addComment = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/3vnHMFG69E4WPq0bdrAl/comments', {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: itemId,
+      username: userName,
+      comment: userComment,
+    }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+  const commentText = await addComment.text();
+  return commentText;
+};
```

Comment:
- [ ] As requested by the previous reviewer [here](https://github.com/teitonny/Javascript-capstone-project/pull/31/files#r1103626334) and [here](https://github.com/teitonny/Javascript-capstone-project/pull/31/files#r1103620709), kindly avoid repeating yourself by minimizing the number of lines of code :recycle: